db/migrate/20160912000001_set_default_run_only_when_new_commit.rb

```diff
@@ -0,0 +1,5 @@
+class SetDefaultRunOnlyWhenNewCommit < ActiveRecord::Migration
+  def change
+    change_column :crons, :run_only_when_new_commit, :boolean, :default => false
```

Comment:
There is data already for the column that needs to be changed. The default value for the column was `true` earlier, which is why we need a migration to set the constraint to `default: false`.

This is because we are trying to simplify this option. Earlier it was confusing to read and interpret behaviour.

(It might sound confusing why we completely flip the default values here. I am still checking with Josh how to handle this change for the users.)